BastionUserOnboarding-SSH-over-SSM.md

```diff
@@ -163,14 +174,18 @@ Host ssh.bastion-prod.probation.hmpps.dsd.io moj_prod_bastion awsprodgw
   ForwardAgent yes
   User YOUR_USER_NAME_HERE
   IdentityFile ~/.ssh/moj_prod_rsa
-  ProxyCommand none
+  ProxyCommand sh -c "aws ssm start-session --target PROD_BASTION_INSTANCE_ID --profile ENG_PROD_PROFILE_NAME --document-name AWS-StartSSHSession --parameters 'portNumber=%p'"
 ```
 
 * Windows
 
-Replace `USER_NAME` with your ssh username. This is usually `<first initial><surname>`, e.g. jbloggs. Also replace `HOMEDIR` with the path to the home directory in Windows (e.g. `C:\Users\Joe.Bloggs`). 
-
-Note: this example assumes the user doesn't have any pre-existing SSH config.
+Replace `USER_NAME` with your ssh username. This is usually `<first initial><surname>`, e.g. jbloggs. <br />
+Replace `HOMEDIR` with the path to the home directory in Windows (e.g. `C:\Users\Joe.Bloggs`).<br />
+Replace `PROD_BASTION_INSTANCE_ID` with the instance id. You can retrieve the dev bastion instance id by running
```

Comment:
2nd sentence of this line mentions dev rather than prod

Reply:
thanks Pete, resolved it